src/Compilers/VisualBasic/Test/Semantic/Semantics/RequiredMembersTests.vb

```diff
@@ -306,8 +318,12 @@ Module M
     End Sub
 End Module"
 
-            Dim comp = CreateCompilation(vbCode, {cComp.EmitToImageReference()})
-            comp.AssertNoDiagnostics
+            Dim comp = CreateCompilation(vbCode, {cComp.EmitToImageReference()}, targetFramework:=TargetFramework.Net70)
```

Comment:
Turns out some of these compilations were not targeting a version of .NET with `SetsRequiredMembers` in it, which was then giving missing symbol text in `ToTestDisplayString()`.